recipes/automated-images/sdk/v2018.2/logs/generate_configs.sh-DEBUG-LOG.txt
$ ./generate_configs.sh 
-----------------------------------
Checking for dependencies...
-----------------------------------
Base docker image [found] (ubuntu:16.04.3)
Xilinx SDK Web Installer: [Exists] depends/Xilinx_SDK_2018.2_0614_1954_Lin64.bin
-----------------------------------
Docker Build Context (Working)...
-----------------------------------
DOCKER_INSTALL_DIR=.
DOCKER_BUILD_WORKING_DIR=/xilinx/local/repositories/gitlab/rjmoss/xilinx-docker/recipes/automated-images/sdk/v2018.2
-----------------------------------
-----------------------------------
Launching Python HTTP Server...
-----------------------------------
+ SERVER_PID=385
+ python3 -m http.server
++ ifconfig docker0
++ grep 'inet\s'
++ awk '{print $2}'
+ SERVER_IP=172.17.0.1
+ '[' 1 -ne 0 ']'
+ set +x
SERVER_IP=172.17.0.1
SERVER_PID=385
-----------------------------------
-----------------------------------
Building the Docker Image...
-----------------------------------
DOCKER_CACHE=--no-cache
DOCKER_FILE_NAME=Dockerfile.generate_configs
DOCKER_FILE_STAGE=base_os_depends_v2018.2
DOCKER_IMAGE=dependency_generation:v2018.2
DOCKER_INSTALL_DIR=.
INSTALL_SERVER_URL=172.17.0.1:8000
-----------------------------------
Arguments...
-----------------------------------
        --build-arg USER_ACCT="xilinx"
        --build-arg HOME_DIR="/home/xilinx"
        --build-arg XLNX_INSTALL_LOCATION="/opt/Xilinx"
        --build-arg INSTALL_SERVER_URL="172.17.0.1:8000"
  --build-arg XLNX_XSDK_WEB_INSTALLER="depends/Xilinx_SDK_2018.2_0614_1954_Lin64.bin"
  --build-arg XLNX_XSDK_BATCH_CONFIG_FILE="configs/xlnx_xsdk.config"
  --build-arg XLNX_XSDK_OFFLINE_INSTALLER="depends/Xilinx_SDK_2018.2_0614_1954_Lin64.tar.gz"
  --build-arg BUILD_DEBUG="1"
-----------------------------------
+ docker build --no-cache -f ./Dockerfile.generate_configs --target base_os_depends_v2018.2 -t dependency_generation:v2018.2 --build-arg USER_ACCT=xilinx --build-arg HOME_DIR=/home/xilinx --build-arg XLNX_INSTALL_LOCATION=/opt/Xilinx --build-arg INSTALL_SERVER_URL=172.17.0.1:8000 --build-arg BUILD_DEBUG=1 .
Sending build context to Docker daemon  8.192kB
Serving HTTP on 0.0.0.0 port 8000 (http://0.0.0.0:8000/) ...
Step 1/10 : FROM ubuntu:16.04.3 AS base_os_depends_v2018.2
 ---> 8ce789b819ee
Step 2/10 : LABEL author="Jason Moss"
 ---> Running in 6b226767f020
Removing intermediate container 6b226767f020
 ---> d086530a1368
Step 3/10 : ENV DEBIAN_FRONTEND=noninteractive
 ---> Running in 1ffc8cad622b
Removing intermediate container 1ffc8cad622b
 ---> bce326ad2336
Step 4/10 : ARG BUILD_DEBUG
 ---> Running in f4693008fd68
Removing intermediate container f4693008fd68
 ---> 82bc1fe0399e
Step 5/10 : RUN if [ $BUILD_DEBUG -ne 0 ]; then set -x; fi      && echo "APT::Install-Recommends false;" > /etc/apt/apt.conf.d/00norecommends   && echo "APT::AutoRemove::RecommendsImportant false;" >> /etc/apt/apt.conf.d/00norecommends         && echo "APT::Install-Suggests false;" > /etc/apt/apt.conf.d/00nosuggests       && echo "APT::AutoRemove::SuggestsImportant false;" >> /etc/apt/apt.conf.d/00nosuggests     && apt-get update       && apt-get install -y           apt-utils               vim     && apt-get install locales      && locale-gen en_US.UTF-8   && dpkg-reconfigure             --frontend $DEBIAN_FRONTEND locales
 ---> Running in 61450d52e820
+ echo APT::Install-Recommends false;
+ echo APT::AutoRemove::RecommendsImportant false;
+ echo APT::Install-Suggests false;
+ echo APT::AutoRemove::SuggestsImportant false;
+ apt-get update
Get:1 http://security.ubuntu.com/ubuntu xenial-security InRelease [109 kB]
Hit:2 http://archive.ubuntu.com/ubuntu xenial InRelease
Get:3 http://archive.ubuntu.com/ubuntu xenial-updates InRelease [109 kB]
Get:4 http://archive.ubuntu.com/ubuntu xenial-backports InRelease [107 kB]
Get:5 http://security.ubuntu.com/ubuntu xenial-security/main amd64 Packages [902 kB]
Get:6 http://archive.ubuntu.com/ubuntu xenial/universe amd64 Packages [7532 kB]
Get:7 http://security.ubuntu.com/ubuntu xenial-security/main Translation-en [336 kB]
Get:8 http://security.ubuntu.com/ubuntu xenial-security/restricted amd64 Packages [7204 B]
Get:9 http://security.ubuntu.com/ubuntu xenial-security/restricted Translation-en [2152 B]
Get:10 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 Packages [496 kB]
Get:11 http://security.ubuntu.com/ubuntu xenial-security/universe Translation-en [203 kB]
Get:12 http://security.ubuntu.com/ubuntu xenial-security/multiverse amd64 Packages [6088 B]
Get:13 http://security.ubuntu.com/ubuntu xenial-security/multiverse Translation-en [2888 B]
Get:14 http://archive.ubuntu.com/ubuntu xenial/universe Translation-en [4354 kB]
Get:15 http://archive.ubuntu.com/ubuntu xenial/multiverse amd64 Packages [144 kB]
Get:16 http://archive.ubuntu.com/ubuntu xenial/multiverse Translation-en [106 kB]
Get:17 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 Packages [1178 kB]
Get:18 http://archive.ubuntu.com/ubuntu xenial-updates/main Translation-en [442 kB]
Get:19 http://archive.ubuntu.com/ubuntu xenial-updates/restricted amd64 Packages [7576 B]
Get:20 http://archive.ubuntu.com/ubuntu xenial-updates/restricted Translation-en [2272 B]
Get:21 http://archive.ubuntu.com/ubuntu xenial-updates/universe amd64 Packages [800 kB]
Get:22 http://archive.ubuntu.com/ubuntu xenial-updates/universe Translation-en [335 kB]
Get:23 http://archive.ubuntu.com/ubuntu xenial-updates/multiverse amd64 Packages [17.1 kB]
Get:24 http://archive.ubuntu.com/ubuntu xenial-updates/multiverse Translation-en [8632 B]
Get:25 http://archive.ubuntu.com/ubuntu xenial-backports/main amd64 Packages [7280 B]
Get:26 http://archive.ubuntu.com/ubuntu xenial-backports/main Translation-en [4456 B]
Get:27 http://archive.ubuntu.com/ubuntu xenial-backports/universe amd64 Packages [8316 B]
Get:28 http://archive.ubuntu.com/ubuntu xenial-backports/universe Translation-en [4476 B]
Fetched 17.2 MB in 3s (5098 kB/s)
Reading package lists...
+ apt-get install -y apt-utils vim
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  apt libapt-inst2.0 libapt-pkg5.0 libexpat1 libgpm2 libmpdec2 libpython3.5
  libpython3.5-minimal libpython3.5-stdlib libsqlite3-0 libssl1.0.0
  mime-support vim-common vim-runtime
Suggested packages:
  aptitude | synaptic | wajig dpkg-dev apt-doc python-apt gpm ctags vim-doc
  vim-scripts vim-gnome-py2 | vim-gtk-py2 | vim-gtk3-py2 | vim-athena-py2
  | vim-nox-py2
Recommended packages:
  file
The following NEW packages will be installed:
  apt-utils libapt-inst2.0 libexpat1 libgpm2 libmpdec2 libpython3.5
  libpython3.5-minimal libpython3.5-stdlib libsqlite3-0 libssl1.0.0
  mime-support vim vim-common vim-runtime
The following packages will be upgraded:
  apt libapt-pkg5.0
2 upgraded, 14 newly installed, 0 to remove and 48 not upgraded.
Need to get 14.1 MB of archives.
After this operation, 55.6 MB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libapt-pkg5.0 amd64 1.2.32ubuntu0.1 [713 kB]
Get:2 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 apt amd64 1.2.32ubuntu0.1 [1087 kB]
Get:3 http://archive.ubuntu.com/ubuntu xenial/main amd64 libgpm2 amd64 1.20.4-6.1 [16.5 kB]
Get:4 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libapt-inst2.0 amd64 1.2.32ubuntu0.1 [54.5 kB]
Get:5 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 apt-utils amd64 1.2.32ubuntu0.1 [197 kB]
Get:6 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libexpat1 amd64 2.1.0-7ubuntu0.16.04.5 [71.5 kB]
Get:7 http://archive.ubuntu.com/ubuntu xenial/main amd64 libmpdec2 amd64 2.4.2-1 [82.6 kB]
Get:8 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libssl1.0.0 amd64 1.0.2g-1ubuntu4.16 [1084 kB]
Get:9 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libpython3.5-minimal amd64 3.5.2-2ubuntu0~16.04.11 [525 kB]
Get:10 http://archive.ubuntu.com/ubuntu xenial/main amd64 mime-support all 3.59ubuntu1 [31.0 kB]
Get:11 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libsqlite3-0 amd64 3.11.0-1ubuntu1.5 [398 kB]
Get:12 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libpython3.5-stdlib amd64 3.5.2-2ubuntu0~16.04.11 [2132 kB]
Get:13 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 vim-common amd64 2:7.4.1689-3ubuntu1.4 [103 kB]
Get:14 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libpython3.5 amd64 3.5.2-2ubuntu0~16.04.11 [1360 kB]
Get:15 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 vim-runtime all 2:7.4.1689-3ubuntu1.4 [5169 kB]
Get:16 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 vim amd64 2:7.4.1689-3ubuntu1.4 [1036 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 14.1 MB in 1s (9473 kB/s)
(Reading database ... 4766 files and directories currently installed.)
Preparing to unpack .../libapt-pkg5.0_1.2.32ubuntu0.1_amd64.deb ...
Unpacking libapt-pkg5.0:amd64 (1.2.32ubuntu0.1) over (1.2.24) ...
Processing triggers for libc-bin (2.23-0ubuntu9) ...
Setting up libapt-pkg5.0:amd64 (1.2.32ubuntu0.1) ...
Processing triggers for libc-bin (2.23-0ubuntu9) ...
(Reading database ... 4766 files and directories currently installed.)
Preparing to unpack .../apt_1.2.32ubuntu0.1_amd64.deb ...
Unpacking apt (1.2.32ubuntu0.1) over (1.2.24) ...
Processing triggers for libc-bin (2.23-0ubuntu9) ...
Setting up apt (1.2.32ubuntu0.1) ...
Installing new version of config file /etc/apt/apt.conf.d/01autoremove ...
Processing triggers for libc-bin (2.23-0ubuntu9) ...
Selecting previously unselected package libgpm2:amd64.
(Reading database ... 4775 files and directories currently installed.)
Preparing to unpack .../libgpm2_1.20.4-6.1_amd64.deb ...
Unpacking libgpm2:amd64 (1.20.4-6.1) ...
Selecting previously unselected package libapt-inst2.0:amd64.
Preparing to unpack .../libapt-inst2.0_1.2.32ubuntu0.1_amd64.deb ...
Unpacking libapt-inst2.0:amd64 (1.2.32ubuntu0.1) ...
Selecting previously unselected package apt-utils.
Preparing to unpack .../apt-utils_1.2.32ubuntu0.1_amd64.deb ...
Unpacking apt-utils (1.2.32ubuntu0.1) ...
Selecting previously unselected package libexpat1:amd64.
Preparing to unpack .../libexpat1_2.1.0-7ubuntu0.16.04.5_amd64.deb ...
Unpacking libexpat1:amd64 (2.1.0-7ubuntu0.16.04.5) ...
Selecting previously unselected package libmpdec2:amd64.
Preparing to unpack .../libmpdec2_2.4.2-1_amd64.deb ...
Unpacking libmpdec2:amd64 (2.4.2-1) ...
Selecting previously unselected package libssl1.0.0:amd64.
Preparing to unpack .../libssl1.0.0_1.0.2g-1ubuntu4.16_amd64.deb ...
Unpacking libssl1.0.0:amd64 (1.0.2g-1ubuntu4.16) ...
Selecting previously unselected package libpython3.5-minimal:amd64.
Preparing to unpack .../libpython3.5-minimal_3.5.2-2ubuntu0~16.04.11_amd64.deb ...
Unpacking libpython3.5-minimal:amd64 (3.5.2-2ubuntu0~16.04.11) ...
Selecting previously unselected package mime-support.
Preparing to unpack .../mime-support_3.59ubuntu1_all.deb ...
Unpacking mime-support (3.59ubuntu1) ...
Selecting previously unselected package libsqlite3-0:amd64.
Preparing to unpack .../libsqlite3-0_3.11.0-1ubuntu1.5_amd64.deb ...
Unpacking libsqlite3-0:amd64 (3.11.0-1ubuntu1.5) ...
Selecting previously unselected package libpython3.5-stdlib:amd64.
Preparing to unpack .../libpython3.5-stdlib_3.5.2-2ubuntu0~16.04.11_amd64.deb ...
Unpacking libpython3.5-stdlib:amd64 (3.5.2-2ubuntu0~16.04.11) ...
Selecting previously unselected package vim-common.
Preparing to unpack .../vim-common_2%3a7.4.1689-3ubuntu1.4_amd64.deb ...
Unpacking vim-common (2:7.4.1689-3ubuntu1.4) ...
Selecting previously unselected package libpython3.5:amd64.
Preparing to unpack .../libpython3.5_3.5.2-2ubuntu0~16.04.11_amd64.deb ...
Unpacking libpython3.5:amd64 (3.5.2-2ubuntu0~16.04.11) ...
Selecting previously unselected package vim-runtime.
Preparing to unpack .../vim-runtime_2%3a7.4.1689-3ubuntu1.4_all.deb ...
Adding 'diversion of /usr/share/vim/vim74/doc/help.txt to /usr/share/vim/vim74/doc/help.txt.vim-tiny by vim-runtime'
Adding 'diversion of /usr/share/vim/vim74/doc/tags to /usr/share/vim/vim74/doc/tags.vim-tiny by vim-runtime'
Unpacking vim-runtime (2:7.4.1689-3ubuntu1.4) ...
Selecting previously unselected package vim.
Preparing to unpack .../vim_2%3a7.4.1689-3ubuntu1.4_amd64.deb ...
Unpacking vim (2:7.4.1689-3ubuntu1.4) ...
Processing triggers for libc-bin (2.23-0ubuntu9) ...
Setting up libgpm2:amd64 (1.20.4-6.1) ...
Setting up libapt-inst2.0:amd64 (1.2.32ubuntu0.1) ...
Setting up apt-utils (1.2.32ubuntu0.1) ...
Setting up libexpat1:amd64 (2.1.0-7ubuntu0.16.04.5) ...
Setting up libmpdec2:amd64 (2.4.2-1) ...
Setting up libssl1.0.0:amd64 (1.0.2g-1ubuntu4.16) ...
Setting up libpython3.5-minimal:amd64 (3.5.2-2ubuntu0~16.04.11) ...
Setting up mime-support (3.59ubuntu1) ...
Setting up libsqlite3-0:amd64 (3.11.0-1ubuntu1.5) ...
Setting up libpython3.5-stdlib:amd64 (3.5.2-2ubuntu0~16.04.11) ...
Setting up vim-common (2:7.4.1689-3ubuntu1.4) ...
Setting up libpython3.5:amd64 (3.5.2-2ubuntu0~16.04.11) ...
Setting up vim-runtime (2:7.4.1689-3ubuntu1.4) ...
Setting up vim (2:7.4.1689-3ubuntu1.4) ...
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/vim (vim) in auto mode
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/vimdiff (vimdiff) in auto mode
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/rvim (rvim) in auto mode
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/rview (rview) in auto mode
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/vi (vi) in auto mode
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/view (view) in auto mode
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/ex (ex) in auto mode
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/editor (editor) in auto mode
Processing triggers for libc-bin (2.23-0ubuntu9) ...
+ apt-get install locales
Reading package lists...
Building dependency tree...
Reading state information...
The following NEW packages will be installed:
  locales
0 upgraded, 1 newly installed, 0 to remove and 48 not upgraded.
Need to get 3218 kB of archives.
After this operation, 14.0 MB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 locales all 2.23-0ubuntu11.2 [3218 kB]
Preconfiguring packages ...
Fetched 3218 kB in 1s (2931 kB/s)
Selecting previously unselected package locales.
(Reading database ... 7523 files and directories currently installed.)
Preparing to unpack .../locales_2.23-0ubuntu11.2_all.deb ...
Unpacking locales (2.23-0ubuntu11.2) ...
Setting up locales (2.23-0ubuntu11.2) ...
Generating locales (this might take a while)...
Generation complete.
+ locale-gen en_US.UTF-8
Generating locales (this might take a while)...
  en_US.UTF-8... done
Generation complete.
+ dpkg-reconfigure --frontend noninteractive locales
Generating locales (this might take a while)...
  en_US.UTF-8... done
Generation complete.
Removing intermediate container 61450d52e820
 ---> 77813b31d309
Step 6/10 : RUN if [ $BUILD_DEBUG -ne 0 ]; then set -x; fi      && apt-get install -y           dialog          debconf-utils   && echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
 ---> Running in 7fde31e15074
+ apt-get install -y dialog debconf-utils
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  debconf
Suggested packages:
  debconf-doc libterm-readline-gnu-perl libgtk2-perl libnet-ldap-perl perl
  libqtgui4-perl libqtcore4-perl
Recommended packages:
  debconf-i18n
The following NEW packages will be installed:
  debconf-utils dialog
The following packages will be upgraded:
  debconf
1 upgraded, 2 newly installed, 0 to remove and 47 not upgraded.
Need to get 408 kB of archives.
After this operation, 1234 kB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 debconf all 1.5.58ubuntu2 [136 kB]
Get:2 http://archive.ubuntu.com/ubuntu xenial-updates/universe amd64 debconf-utils all 1.5.58ubuntu2 [57.5 kB]
Get:3 http://archive.ubuntu.com/ubuntu xenial/universe amd64 dialog amd64 1.3-20160209-1 [215 kB]
Preconfiguring packages ...
Fetched 408 kB in 0s (490 kB/s)
(Reading database ... 8120 files and directories currently installed.)
Preparing to unpack .../debconf_1.5.58ubuntu2_all.deb ...
Unpacking debconf (1.5.58ubuntu2) over (1.5.58ubuntu1) ...
Setting up debconf (1.5.58ubuntu2) ...
Selecting previously unselected package debconf-utils.
(Reading database ... 8120 files and directories currently installed.)
Preparing to unpack .../debconf-utils_1.5.58ubuntu2_all.deb ...
Unpacking debconf-utils (1.5.58ubuntu2) ...
Selecting previously unselected package dialog.
Preparing to unpack .../dialog_1.3-20160209-1_amd64.deb ...
Unpacking dialog (1.3-20160209-1) ...
Setting up debconf-utils (1.5.58ubuntu2) ...
Setting up dialog (1.3-20160209-1) ...
+ echo debconf debconf/frontend select Noninteractive
+ debconf-set-selections
Removing intermediate container 7fde31e15074
 ---> 2109489b565e
Step 7/10 : ARG USER_ACCT
 ---> Running in b10c6693ba8c
Removing intermediate container b10c6693ba8c
 ---> d5cdbd35c9ef
Step 8/10 : ARG HOME_DIR
 ---> Running in 2880ba25ac28
Removing intermediate container 2880ba25ac28
 ---> 73600192a7bd
Step 9/10 : ARG XLNX_INSTALL_LOCATION
 ---> Running in fd11f8f090d4
Removing intermediate container fd11f8f090d4
 ---> f6d07b8fff0f
Step 10/10 : RUN if [ $BUILD_DEBUG -ne 0 ]; then set -x; fi     && apt-get install -y           sudo    && adduser              --disabled-password             --gecos ''         --shell /bin/bash                --home $HOME_DIR                $USER_ACCT      && echo                 "$USER_ACCT ALL=(ALL) NOPASSWD: ALL"            >> /etc/sudoers         && adduser $USER_ACCT dialout       && adduser $USER_ACCT uucp      && chgrp uucp /var/lock         && chmod g+w /var/lock  && mkdir -p $XLNX_INSTALL_LOCATION      && chown -R $USER_ACCT:$USER_ACCT $XLNX_INSTALL_LOCATION    && echo "dash dash/sh boolean false" | debconf-set-selections   && DEBIAN_FRONTEND=$DEBIAN_FRONTEND dpkg-reconfigure dash
 ---> Running in 5aadf4c6e24c
+ apt-get install -y sudo
Reading package lists...
Building dependency tree...
Reading state information...
The following NEW packages will be installed:
  sudo
0 upgraded, 1 newly installed, 0 to remove and 47 not upgraded.
Need to get 390 kB of archives.
After this operation, 1622 kB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 sudo amd64 1.8.16-0ubuntu1.9 [390 kB]
Fetched 390 kB in 0s (519 kB/s)
Selecting previously unselected package sudo.
(Reading database ... 8339 files and directories currently installed.)
Preparing to unpack .../sudo_1.8.16-0ubuntu1.9_amd64.deb ...
Unpacking sudo (1.8.16-0ubuntu1.9) ...
Setting up sudo (1.8.16-0ubuntu1.9) ...
+ adduser --disabled-password --gecos  --shell /bin/bash --home /home/xilinx xilinx
Adding user `xilinx' ...
Adding new group `xilinx' (1000) ...
Adding new user `xilinx' (1000) with group `xilinx' ...
Creating home directory `/home/xilinx' ...
Copying files from `/etc/skel' ...
+ echo xilinx ALL=(ALL) NOPASSWD: ALL
+ adduser xilinx dialout
Adding user `xilinx' to group `dialout' ...
Adding user xilinx to group dialout
Done.
+ adduser xilinx uucp
Adding user `xilinx' to group `uucp' ...
Adding user xilinx to group uucp
Done.
+ chgrp uucp /var/lock
+ chmod g+w /var/lock
+ mkdir -p /opt/Xilinx
+ chown -R xilinx:xilinx /opt/Xilinx
+ echo dash dash/sh boolean false
+ debconf-set-selections
+ DEBIAN_FRONTEND=noninteractive dpkg-reconfigure dash
Removing 'diversion of /bin/sh to /bin/sh.distrib by dash'
Adding 'diversion of /bin/sh to /bin/sh.distrib by bash'
Removing 'diversion of /usr/share/man/man1/sh.1.gz to /usr/share/man/man1/sh.distrib.1.gz by dash'
Adding 'diversion of /usr/share/man/man1/sh.1.gz to /usr/share/man/man1/sh.distrib.1.gz by bash'
Removing intermediate container 5aadf4c6e24c
 ---> 7b24a9bacc15
[Warning] One or more build-args [INSTALL_SERVER_URL] were not consumed
Successfully built 7b24a9bacc15
Successfully tagged dependency_generation:v2018.2
+ xhost +
access control disabled, clients can connect from any host
+ '[' 1 -ne 0 ']'
+ set +x
-----------------------------------
Create a base docker container...
-----------------------------------
DOCKER_CONTAINER_NAME=
-----------------------------------
+ docker run --name build_xsdk_depends_v2018.2 -v /tmp/.X11-unix:/tmp/.X11-unix -e DISPLAY=:0 -itd dependency_generation:v2018.2 /bin/bash
88c133e7d291387d7c448c81fa22c3cf92fc861ac5556fe7578270d2d35994b2
+ '[' 1 -ne 0 ']'
+ set +x
-----------------------------------
Install support packages...
-----------------------------------

recipes/vivado/v2020.1/include/configuration.sh
#!/bin/bash
########################################################################################
# Docker Image Build Variable Custom Configuration:
#   - Customize definitions of arguments for building images for v2020.1
#	- Used with: Dockerfile.xx_v2020.1
#
# Maintainer:
#	- Jason Moss (jason.moss@avnet.com)
#	- Xilinx Applications Engineer, Embedded Software
#
# Created: 
#	- 6/25/2020
#
#######################################################################################
# Docker Build Script Debug Tracing
########################################################################################
#BUILD_DEBUG=1 Turns shell command expansion on in Docker build scripts
#BUILD_DEBUG=0 Turns shell expandion off in Docker build scripts
BUILD_DEBUG=1

########################################################################################
########################################################################################
# DOCKER FILE GENERAL CONFIGURATION PARAMETERS:
########################################################################################
########################################################################################
# User account information
# USER_ACCT: user account name within docker image
# HOME_DIR : user account home directory
USER_ACCT=xilinx
HOME_DIR=/home/$USER_ACCT

# GIT Configuration
# GIT_USER_NAME: Username for git configuration
# GIT_USER_EMAIL: email address for git configuration
GIT_USER_NAME="Xilinx User"
GIT_USER_EMAIL="Xilinx.User@dummyaddress.com"

# Xilinx Release Information
XLNX_RELEASE_VERSION=v2020.1

# Xilinx tool information
XLNX_TOOL_INFO=vivado
XLNX_TOOL_INSTALLER_NAME=Vivado

# Docker File Recipe Name
# Vivado - possible stage assignments
#DOCKER_FILE_NAME=Dockerfile.$XLNX_TOOL_INFO_$XLNX_RELEASE_VERSION
#DOCKER_FILE_STAGE=base_os_$XLNX_TOOL_INFO"_"$XLNX_RELEASE_VERSION
#DOCKER_FILE_STAGE=xilinx_install_depends_$XLNX_TOOL_INFO"_"$XLNX_RELEASE_VERSION
#DOCKER_FILE_STAGE=xilinx_install_$XLNX_TOOL_INFO"_"$XLNX_RELEASE_VERSION
DOCKER_FILE_STAGE=xilinx_install_$XLNX_TOOL_INFO"_"$XLNX_RELEASE_VERSION

# Docker Image Name
#DOCKER_IMAGE_NAME=xilinx-yocto
#DOCKER_IMAGE_NAME=xilinx-vivado
#DOCKER_IMAGE_NAME=xilinx-petalinux
DOCKER_IMAGE_NAME=xilinx-$XLNX_TOOL_INFO

# Docker base OS Images
DOCKER_BASE_OS=ubuntu
DOCKER_BASE_OS_TAG=18.04.2

# Should Docker use Cache when building?
# - A couple of important reasons to DISABLE the use of the cache
#    - 1) You want to ensure you can build the image completey from scratch
#    - 2) The image build is failiing on an APT stage fetch
#         - Updates are made to repositories regularly and your apt-cache in the cached image may be stale
#         - This causes apt-get instructions to fail because the system has outdated information about a package
# To use cached images, set DOCKER_CACHE= ''
# To force rebuild, set DOCKER_CACHE='--no-cache'
# DOCKER_CACHE='--no-cache'
# Turn off use of cached images
#DOCKER_CACHE=''
DOCKER_CACHE='--no-cache'

# Location the build is executed from
DOCKER_BUILD_WORKING_DIR=`pwd`
DOCKER_BUILD_TMPDIR=$DOCKER_BUILD_WORKING_DIR/tmp

# Docker build image parameters
DOCKER_IMAGE_VERSION=$XLNX_RELEASE_VERSION

########################################################################################
########################################################################################
# Target Path Variables for Docker Image:
########################################################################################
########################################################################################
# Xilinx Tool Install Location
XLNX_INSTALL_LOCATION=/opt/Xilinx

# Xilinx Tool Downloads Location
XLNX_DOWNLOAD_LOCATION=$XLNX_INSTALL_LOCATION/downloads

########################################################################################
########################################################################################
# Host OS Path Variables for Docker build:
########################################################################################
########################################################################################
DOCKER_INSTALL_DIR=.

# Xilinx Configuration and Dependency folder definitions
# These locations must be within the docker build context
# (meaning accessible from the base folder of the Dockerfile)
# INSTALL_CONFIGS_DIR: folder in docker context on host for
#	build related configuration files for headless install
# INSTALL_DEPENDS_DIR:  folder in docker context on host for
#	build related installers/downloads used in installation
INSTALL_CONFIGS_DIR=configs
INSTALL_DEPENDS_DIR=depends

########################################################################################
########################################################################################
# Depdendency / Configuration File Related Variables
########################################################################################
########################################################################################
# DOCKER_BUILD_XLNX_MALI_LOCAL:
# '0' = The docker build script will not construct XLNX_MALI_URL
#       using the python server address, but will pass the URL setup above
#
# '1' = The docker build script will construct XLNX_MALI_URL
#		using the python server address and the INSTALL_DEPENDS_DIR variable
DOCKER_BUILD_XLNX_MALI_LOCAL='1'

# Xilinx ARM Mali Pre-built binaries
# XLNX_MALI_URL: Base download URL
# XLNX_MALI_BINARY: filename to download
# **Downloading Direct from Xilinx no longer works due to click-through license requirements
# Set #2: Download from local archive by use of python http.server (address set later in this script)
# DOCKER_BUILD_XLNX_MALI_LOCAL=1
XLNX_MALI_URL=0.0.0.0:8000
XLNX_MALI_BINARY=$INSTALL_DEPENDS_DIR/mali-400-userspace.tar
#XLNX_MALI_BINARY=$INSTALL_DEPENDS_DIR/mali-400-userspace-with-android-2019.1.tar

# Configuration Files for batch mode installation
# KEYBOARD_CONFIG_FILE:	Keyboard setting configuration file
#	for headless selection of the keyboard setup
#	Required for the X-based XSDK installer package
KEYBOARD_CONFIG_FILE=$INSTALL_CONFIGS_DIR/keyboard_settings.conf

# Configuration file for xterm sessions inside of the docker container
# XTERM_CONFIG_FILE: XTerm session configuratino file
#	Changes color scheme and font to something more readable (default is white background)
#   Changes scrollback to 1 million lines and enables the scroll bar
#   Notes on Copy-Paste with the host:
#	 	Copy from Host to XTerm:
#			- Host may copy with "CTRL-C" or "Right-Click->Copy"
#			- Use center mouse button (scroll wheel) to paste into XTerm session
#
#		Copy from XTerm to Host:
#			- Select text in XTerm session to copy using cursor (left-click and drag over text to copy)
#			- Host may paste with center mouse button (scroll wheel)
#			- Host clipboard contents from host copy maintained in separate buffer (CTRL-V)
#-------------
XTERM_CONFIG_FILE=$INSTALL_CONFIGS_DIR/XTerm

# Configuration Files for minicom session inside of the docker container
# MINICOM_CONFIG_FILE: Minicom default settings configuration file
#   115200-8-N-1, no hardware flow control by default
MINICOM_CONFIG_FILE=$INSTALL_CONFIGS_DIR/.minirc.dfl

# Xilinx Vivado batch mode: Batch mode install configuration template
#	for headless install of Xilinx SDK components
# xlnx_vivado_system_edition = Vivado System Edition installation
# ??? = Vivado ??? Edition installation
XLNX_VIVADO_BATCH_CONFIG_FILE=$INSTALL_CONFIGS_DIR/xlnx_vivado_system_edition.config

# Xilinx Vivado Web-based installer
# XLNX_XSDK_WEBINSTALLER: Official web-based installer for the SDK release
XLNX_VIVADO_WEB_INSTALLER=$INSTALL_DEPENDS_DIR/Xilinx_Unified_2020.1_0602_1208_Lin64.bin

# Xilinx Vivado Pre-downloaded offline install bundle
# - This is downloaded and created by:
#   1. Running the web-installer with the batch mode configuration
#   2. Downloading the files for offline install
#   3. Manually archiving files in a tarball
XLNX_VIVADO_OFFLINE_INSTALLER=$INSTALL_DEPENDS_DIR/Xilinx_Unified_2020.1_0602_1208_Lin64.bin.tar.gz

# Local Python3 http server to transfer files into container
# INSTALL_SERVER_URL: Set automatically later in this script when
#	http server is spawned for file transfer to container
INSTALL_SERVER_URL=0.0.0.0:8000

########################################################################################
########################################################################################

# Define File system error code
EX_OSFILE=72




